### DISCUSSION TITLE
PCT combos suggestions

### DISCUSSION THREAD

**Discussion by username4447** [PrincessRTFM/XIVComboPlugin#109] [General]
">" means "becomes"

in addition to what is already in the plugin (fire combo > cyan combo)

weapon motif > steel muse > hammer stamp combo
landscape motif > scenic muse > star prism
creature motif > living muse
holy in white > comet in black (when comet is up)

![image](https://github.com/user-attachments/assets/5e1c15b3-44ec-482f-bba4-e8587849e7c7)

**Comment by VariableVixen** [comment]
Pinging @FrigidWalrus for this, because they're the one who's been working on PCT stuff so far

**Comment by FrigidWalrus** [comment]
I'll pick at this later today and see what I can do.

**Reply by FrigidWalrus** [reply]
I've worked out a working method with the Weapon Motif/Hammer Combo part, so I should be able to work up the rest. Need sleep though. I'll get to work on this more later today.

**Reply by FrigidWalrus** [reply]
One thing to note. Striking Muse won't actually allow you to use it until you're already in combat. I'm going to guess the rest of the muse types are the same, so you'll be able to draw the motif out of combat, and the muse will show greyed out until you're actually in combat.

**Reply by username4447** [reply]
> One thing to note. Striking Muse won't actually allow you to use it until you're already in combat. I'm going to guess the rest of the muse types are the same, so you'll be able to draw the motif out of combat, and the muse will show greyed out until you're actually in combat.

It is only an issue for scenic muse, but not really because in the opener :
You start by drawing the 3 motifs instant out of combat, then hardcast a 4sec GCD into steel muse and after that you're considered in fight so there's no issue, + you're still doing two GCDs before buffing

**Reply by FrigidWalrus** [reply]
Starry muse was easy, as it works off much the same logic as the hammer combo line. Holy in White to Comet in Black was also fairly simple to implement. 

Creature combo is much more difficult, because there's several different creature motifs, and each has its own muse skill, with multiple charges. There are definitely times where you would want to just draw the next motif to progress through motifs as opposed to using the current muse, so not sure if I would add that into the combo itself, honestly. I can see about maybe detecting if Living Muse !has charges, then show the next motif skill, but It seems like a little too much like taking away options, and too little gain in comboing it.

**Reply by FrigidWalrus** [reply]
Took another look, after looking at it for nearly 30 minutes, and realized how to implement it. I didn't realize using a muse makes that motif go away and not be usable until a new motif is drawn.

**Reply by FrigidWalrus** [reply]
The update has been pushed out. all the requested combos _should_ work, but keep in mind I am at level 80 PCT, so I can't actually test Comet or the full hammer combo. The logic checks out, though, and it shares some logic used in other places, so it should work correctly. 


_Should._

**Comment by username4447** [comment]
4201, 4202, 4203, 4204, 4205, 4206 and 4208 works like a charm I confirm and I thank you a lot

**EDIT :** there's something weird with the comet in black and sub palette (4207)
steps to reproduce :
1. have 50 gauge OR starry muse additional effect allowing you one free use of sub palette
2. use sub palette
3. do the cyan combo cyan > yellow > **magenta**
4. holy in white is no longer replaced by comet in black. It's greyed instead

once magenta is done, it gives a holy in white.

![image](https://github.com/user-attachments/assets/68d310f4-1804-4c5f-b473-69ed1ff6d87c)


At this exact moment, holy in white is not replaced by comet in black
this forces us to use comet in black before ending the cyan > yellow > magenta combo, which is not possible in the opener

It is the same for the AOE combo

**Reply by VariableVixen** [reply]
Frigid, our PCT dev, will look into that once they're available again, but please use the debug IDs rather than the ordinal numbers for reports. The numbers on the list are generated by the UI window and aren't constant, let alone present anywhere in the codebase. The debug IDs, on the other hand, are both of those things, which allows us to easily locate the preset in question.

**Reply by username4447** [reply]
edited

**Comment by FrigidWalrus** [comment]
I've changed the thing I think was causing the issue, but as I don't have PCT at 90, I won't be able to fiddle with it and test it. This might need to wait until I can get PCT to 90 to figure out exactly what is going on there. It SHOULD now just check that you've got inverted colors. It was looking for the hyperphantasia stack, but technically it doesn't need it to be used. 

If this doesn't work, we'll need to close this for now, cause getting PCT to 90 is on The List of Things To Do.

**Reply by username4447** [reply]
It is fixed \o/

**Comment by FrigidWalrus** [comment]
Awesome, glad we could get it working properly. If you ever have more suggestions, feel free to bring them up!

**Reply by VariableVixen** [reply]
Although, please do actually use an issue for them instead of a discussion - those are where the other feature requests go, and they can actually be marked as completed.
